glut/Makefile.am

```diff
@@ -67,8 +67,7 @@ endif
 
 test_glut_SOURCES=test_glut.c
 test_glut_LDADD=$(addprefix $(top_builddir)/, $(LIBS_MGH)) $(LIBS_VXL) -lz \
-	$(LIB_JPEG) $(LIB_TIFF)	$(X_LIBS) $(LIBS_OPENGL) $(top_builddir)/glut/libglut.a
-
+	$(LIB_JPEG) $(LIB_TIFF)	$(top_builddir)/glut/libglut.a $(X_LIBS) $(LIBS_OPENGL) 
```

Comment:
Ok it looks like its ignoring the matrix and the complete set of tests is simply running in parallel. 'make check' will surely exceed the 50min alot allowed by travis for a job:

  https://docs.travis-ci.com/user/customizing-the-build#Build-Timeouts

Either we need to get the matrix of tests working, or only a small subset of freesurfer tests can be integrated into the travis build.